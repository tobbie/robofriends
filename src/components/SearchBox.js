import React from 'react'

const SearchBox = ({searchField, searchChange}) => {
  
    return(
  <div>
      <input 
      aria-label='Search Robots'
      type='search' 
      placeholder='search robots'  
      className='pa3 ba b--green bg-lightest-blue' 
      onChange= { searchChange } 
       />
  </div>
  )
}

export default SearchBox;