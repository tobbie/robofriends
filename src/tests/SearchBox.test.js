import { shallow } from "enzyme";
import React from "react";
import SearchBox from "../components/SearchBox";

describe("searchbox tests", () => {
  it("expect to render searchbox component", () => {
    expect(shallow(<SearchBox />)).toMatchSnapshot();
  });
});
