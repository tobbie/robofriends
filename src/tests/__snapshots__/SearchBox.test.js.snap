// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`searchbox tests expect to render searchbox component 1`] = `ShallowWrapper {}`;
